components/audit_table/user_row/user_update_active_session_row.tsx

```diff
@@ -66,5 +66,6 @@ export default function UserUpdateActiveSessionRow({
             showUserId={showUserId}
             showIp={showIp}
             showSession={showSession}
-        />);
+        />
+    );
```

Comment:
All those small changes in return statements from render is to appease the errors from `indent` rule

Reply:
That's nice to see actually. I always want to go in and change these so that the closing bracket is on the same level as the opening one